interface/src/app/signature/signature.tpl.html

```diff
@@ -5,12 +5,25 @@
 
 <h3 class="text-warning" ng-bind="ctrl.statusMessage"></h3>
 
+<h4>
```

Comment:
I really think this should just be a paragraph `<p>` block. The contents here are not a (short, succinct) section header, which is what an `<h4>` should be.